src/pages/p2p/components/_dp2p.js

```diff
@@ -106,8 +106,14 @@ const StyledIFrame = styled.iframe`
     width: 560px;
     height: 315px;
 
+    @media ${device.tablet} {
+        max-width: 90%;
+    }
+    @media ${device.tabletS} {
+        max-width: 90%;
+    }
     @media ${device.mobileM} {
-        width: 310px;
+        max-width: 90%;
```

Comment:
I can't fully understand the implementation here. If your intention is to have the max-width of the player to be 560px but when shrunk down to for e.g. tablet to maintain 90% width of the screen. You can do something like this instead:
```
const StyledIFrame = styled.iframe`
    height: 315px;
    width: 100%;
    max-width: 560px;

    @media ${device.tablet} {
        width: 90%;
    }
```